File: fase06/ex04/somaPares.js
function somaPares(x1,y1) {

x = Math.ceil(x1)
y = Math.floor(y1)

t=0

for(i=x;i<=y1;i++) {
   if (i/2==Math.floor(i/2)) {
        t+=i
    }
}
return t
}

File: python/poesiaVogon/fase06/ex04/somaPares.py
def somaPares(x1,y1):
    import math
    x = math.ceil(x1)
    y = math.floor(y1)
    
    t=0
    
    for i in range(x,y+1):
       if (i/2==math.floor(i/2)):
            t=i+t
    
    return t

